CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+### v2.0.1
+
+* Support for the Distance Matrix has been added, this replaces the distance API.
```

Comment:
Please add links to the corresponding tickets like in previous versions. Also "Distance API".